### PR TITLE
add support for syncing to specific repos

### PR DESCRIPTION
**Pull request by rrennick**
This PR adds 

- support for specifying syncing to one or more specific repos via the command line
- spacing to the terminal output 
- lists the repos as they are processed to aid in troubleshooting when an error occurs

### Testing

- Run the command against an approved or test repo (could be created under a personal account)
```
npm start -- --dry-run --repo organization/repository1
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ For more details about GitHub acess tokens, check ["Creating a personal access t
 |---------------------|------------------------------------------------------------|
 | `--dry-run`         | Test mode, no changes will happens in any repository       |
 | `--preserve-labels` | New labels introduced directly in GitHub will be preserved |
+| `--repo`            | Sync to this repository. Multiple repositories supported.
```
Comment (claudiosanches) [inline README.md]:
```suggestion
| `--repo`            | Sync to this repository. Multiple repositories supported.    |
```

**Review by claudiosanches — CHANGES_REQUESTED** [review]
Works fine, there's just a minor issue in the README.md

**Comment by rrennick** [comment]
@claudiosanches Thanks for catching that. I committed your suggestion.

**Review by claudiosanches — APPROVED** [review]
Great work!

**Pull request merged**